bookdown/03-optimization.Rmd

```diff
@@ -231,6 +231,231 @@ Note that we do not expect any differences here compared to the non-tuned approa
 * the tuning budget (10 evaluations) is small
 * `r cran_pkg("rpart")` does not benefit that much from tuning
 
+### Tuning with Hyperband {#hyperband}
+
+Besides the more traditional tuning methods listed above, the ecosystem around `r mlr_pkg("mlr3")` offers another procedure for hyperparameter optimization called Hyperband implemented in the `r gh_pkg("mlr-org/mlr3hyperband")` package.
+Hyperband is a budget-oriented procedure, weeding out suboptimally performing configurations early on during a partially sequential training process, increasing tuning efficiency as a consequence.
+For this, a combination of incremental resource allocation and early stopping is used, increasing computational resources of more promising configurations plus terminating lesser ones the further training stages are reached.
+To give an introductional analogy, imagine two horse trainers are given eight untrained horses.
+Both trainers want to end up with the quickest horse to win the incoming race, but they are only given a fixed amount of budget (32 units) to buy food.
```

Comment:
"Both trainers [...] to win the **upcoming** race, but they are only given a fixed budget of 32 units to buy food. "